plugins/javadoc/src/test/kotlin/org/jetbrains/dokka/javadoc/JavadocModuleTemplateMapTest.kt

```diff
@@ -77,4 +78,65 @@ internal class JavadocModuleTemplateMapTest : AbstractJavadocTemplateMapTest() {
             assertEquals(DRI("com.test.package1"), assertIsInstance<RowJavadocListEntry>(list[1]).link.dri.single())
         }
     }
+
+    @Test
+    fun `single class with package documentation (kotlin)`() {
+        testTemplateMapInline(
+            query =
+            """
+            /src/module.md
+            # Module module1
+            ABC
+            
+            /src/source0.kt
+            package com.test.package0
+            class Test
+            """,
+            configuration = config.copy(
+                sourceSets = config.sourceSets.map { sourceSet ->
+                    sourceSet.copy(
+                        includes = setOf(File("src/module.md").also {println(it.absolutePath)})
```

Comment:
Leftover from debugging? Could be confusing to see a random path printed out of the blue otherwise

```suggestion
                        includes = setOf(File("src/module.md"))
```

Reply:
Yes, it was a leftover from debugging. Removed